index.html

```diff
@@ -4,29 +4,55 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="icon" type="image/x-icon" href="./zuru-logo.svg">
+    <link rel="icon" type="image/x-icon" href="./image/zuru-logo.svg">
     <link rel="stylesheet" href="./style.css">
     <title>ZURU</title>
 </head>
 
 <body>
-    <h3 class="heading">ZURU</h3>
+    <div class="app-content">
+        <h3 class="heading">ZURU</h3>
+        <!-- user container -->
```

Comment:
@raaj-oo7 This comment is not needed as the class name already pointing to the reference container itself
Give comments only when needed for example complex logic or block of code that is difficult to understand at first glance